x-pack/test/api_integration/apis/ml/calculate_model_memory_limit.ts

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+const COMMON_HEADERS = {
+  'kbn-xsrf': 'some-xsrf-token',
+};
+
+const testDataList = [
```

Comment:
Could you add tests for:
- 0 metrics, no split field, and 0 influencers (for the simple count detector use case)
- 1 metric, 1 split field, 1 influencer (which is the same as the split field)

Reply:
https://github.com/elastic/kibana/pull/54557/commits/01da7dfb83204a6262e3d5a8cd19151a8352af95